sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/TestHelper.java

```diff
@@ -63,6 +63,6 @@ private static boolean shouldServiceVersionBeTested(ConfigurationServiceVersion
         }
         String[] configuredServiceVersionList = SERVICE_VERSION_FROM_ENV.split(",");
         return Arrays.stream(configuredServiceVersionList).anyMatch(configuredServiceVersion ->
-            serviceVersion.toString().equals(configuredServiceVersion.trim()));
+            serviceVersion.getVersion().equals(configuredServiceVersion.trim()));
```

Comment:
This won't work - you're comparing a ServiceVersion enum value to a string.

Reply:
I am comparing serviceVersion string value to env value.
If say,
```
    V7_0("7.0"),
    V7_1_PREVIEW("7.1-preview");
```
`set AZAZURE_KEYVAULT_TEST_CERTIFICATE_SERVICE_VERSION = 7.0` works